include/meta/meta_libcxx_workaround.hpp

```diff
@@ -0,0 +1,159 @@
+{
+    namespace detail
+    {
+        template <typename T, typename A = std::allocator<T>>
+        using std_list = std::list<T, A>;
+        template <typename T, typename A = std::allocator<T>>
+        using std_forward_list = std::forward_list<T, A>;
+        template <typename T, typename A = std::allocator<T>>
+        using std_vector = std::vector<T, A>;
+        template <typename T, typename A = std::allocator<T>>
+        using std_deque = std::deque<T, A>;
+        template <typename T, typename C = std::char_traits<T>,
+                  typename A = std::allocator<T>>
+        using std_basic_string = std::basic_string<T, C, A>;
+        template <typename K, typename V, typename C = std::less<K>,
+                  typename A = std::allocator<std::pair<K const, V>>>
+        using std_map = std::map<K, V, C, A>;
+        template <typename K, typename V, typename C = std::less<K>,
+                  typename A = std::allocator<std::pair<K const, V>>>
+        using std_multimap = std::multimap<K, V, C, A>;
+        template <typename K, typename C = std::less<K>,
+                  typename A = std::allocator<K>>
+        using std_set = std::set<K, C, A>;
+        template <typename K, typename C = std::less<K>,
+                  typename A = std::allocator<K>>
+        using std_multiset = std::multiset<K, C, A>;
+        template <typename K, typename V, typename H = std::hash<K>,
+                  typename C = std::equal_to<K>,
+                  typename A = std::allocator<std::pair<K const, V>>>
+        using std_unordered_map = std::unordered_map<K, V, H, C, A>;
+        template <typename K, typename V, typename H = std::hash<K>,
+                  typename C = std::equal_to<K>,
+                  typename A = std::allocator<std::pair<K const, V>>>
+        using std_unordered_multimap = std::unordered_multimap<K, V, H, C, A>;
+        template <typename K, typename H = std::hash<K>,
+                  typename C = std::equal_to<K>, typename A = std::allocator<K>>
+        using std_unordered_set = std::unordered_set<K, H, C, A>;
+        template <typename K, typename H = std::hash<K>,
+                  typename C = std::equal_to<K>, typename A = std::allocator<K>>
+        using std_unordered_multiset = std::unordered_multiset<K, H, C, A>;
+        template <typename T, typename C = std_deque<T>>
+        using std_queue = std::queue<T, C>;
+        template <typename T, typename C = std_vector<T>,
+                  class D = std::less<typename C::value_type>>
+        using std_priority_queue = std::priority_queue<T, C, D>;
+        template <typename T, typename C = std_deque<T>>
+        using std_stack = std::stack<T, C>;
+    }
+
+    template <> struct quote< ::std::list > : quote<detail::std_list>
+    {
+    };
+    template <>
+    struct quote< ::std::forward_list > : quote<detail::std_forward_list>
+    {
+    };
+    template <> struct quote< ::std::vector > : quote<detail::std_vector>
+    {
+    };
```

Comment:
Oops, looks like I forgot the specialization for `std::deque`. Need that one, too.